collections/Laud_Misc/MS_Laud_Misc_636.xml

```diff
@@ -37,43 +59,195 @@
+                           Other manuscripts copied by this scribe: corrections in Cambridge, Corpus Christi College 134 (Bishop 1953, p. 440); London, Society of Antiquaries, MS 60, fols 6-71. (Bishop 1953, p. 440) 
+                     </handNote>
+                     <handNote script="textualisNorthern" scope="minor">The addition on fols. 86v-90v in textualis.</handNote>
+                  </handDesc>
                   <decoDesc>
-                     <decoNote type="decInit">Good initial. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 88)</decoNote>
+                     <decoNote type="decInit">Ornamental arabesque initial <q type="character">B</q> on fol. 1r, green
+                        with decoration in red.(<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 88)</decoNote>
+                     <decoNote>Years in red; first letter of each annal usually stroked with red.</decoNote>
+                     <decoNote type="colInit">Plain coloured red initials from fol. 83v onwards (1124) and in the added text on fols. 86v-90v.</decoNote>
                   </decoDesc>
+                  <additions>
+                     <p>'The occasional marginalia in Latin and <foreign>notas</foreign> of s. xiii/xiv (e.g. on f. 18) show that the OE text was read and to some extent understood at this date by someone with an interest in local affairs.' (Ker)</p>
+                  </additions>
+                  
+                  <bindingDesc>
+                     <binding notAfter="1639" notBefore="1636">
+                        <p>Standard binding of the Laudian collection, calf over pasteboard with Laud's arms in gilt; rebacked.</p>
+                     </binding>
+                  </bindingDesc>
+                  <accMat>
+                     <p>Paper leaves were added to the manuscript at the beginning and end, and between the leaves of the manuscript, in the early seventeenth century. Fols 92r-97r contain transcripts
+                        by William Lisle  from the <title>The Parker
+                           Chronicle</title> (CCCC
+                              173, <title>Anglo-Saxon Chronicle</title>, A version). The
+                        quire also contains seventeenth-century tables of contents of nine
+                        Parkerian homilaries and miscellaneous notes in various hands (O'Brien O'Keeffe 2003, p. 41). </p>
+                  </accMat>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1300" notBefore="1121">c. 1121–31; additions c. 1154 and 13th century, second half</origDate>
+                     <origDate calendar="Gregorian" notBefore="1121" notAfter="1131" cert="high">c. 1121–31</origDate>; 
+                     <origDate calendar="Gregorian" when="1155" cert="high">continued c. 1155</origDate>; 
```

Comment:
What will the system do with multiple `<origDate>` elements? Should `<accMat>` be another `<msPart>`?